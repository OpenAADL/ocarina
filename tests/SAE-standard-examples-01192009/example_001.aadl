--test 4.2 Packages draft-01192009

package Aircraft::Cockpit
public
with Avionics::DataTypes, Safety_Properties;

-- bad syntax
--  AirData renames Avionics::DataTypes::AirData;
-- correction
AirData renames data Avionics::DataTypes::AirData;

system MFD
features
Airdata: in data port AirData;
properties
Safety_Properties::Safety_Criticality => high;
end MFD;
end Aircraft::Cockpit;