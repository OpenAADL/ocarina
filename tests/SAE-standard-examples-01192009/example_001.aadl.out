example_001.aadl:5:01:  (package name) is not a package or a property set visible or existing
example_001.aadl:5:27: Safety_Properties (identifier) is not a package or a property set visible or existing
Cannot analyze AADL specifications
